groups.sql

```diff
@@ -0,0 +1,331 @@
+BEGIN TRANSACTION;

```

Comment:
Can you explain what `BEGIN TRANSACTION` does?

Reply:
it's just a side effect of the built-in function of importing from csv-table.